isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -1827,17 +1842,19 @@ namespace isobus
 
 							if (CurrentObjectPoolUploadState::Uninitialized == currentObjectPoolState)
 							{
+								bool transmitSuccessful;
+
 								if (!objectPools[i].uploaded)
 								{
-									bool transmitSuccessful = CANNetworkManager::CANNetwork.send_can_message(static_cast<std::uint32_t>(CANLibParameterGroupNumber::ECUtoVirtualTerminal),
-									                                                                         nullptr,
-									                                                                         objectPools[i].objectPoolSize + 1, // Account for Mux byte
-									                                                                         myControlFunction.get(),
-									                                                                         partnerControlFunction.get(),
-									                                                                         CANIdentifier::CANPriority::PriorityLowest7,
-									                                                                         process_callback,
-									                                                                         this,
-									                                                                         process_internal_object_pool_upload_callback);
+									transmitSuccessful = CANNetworkManager::CANNetwork.send_can_message(static_cast<std::uint32_t>(CANLibParameterGroupNumber::ECUtoVirtualTerminal),
+									                                                                    nullptr,
+									                                                                    objectPools[i].objectPoolSize + 1, // Account for Mux byte
+									                                                                    myControlFunction.get(),
+									                                                                    partnerControlFunction.get(),
+									                                                                    CANIdentifier::CANPriority::PriorityLowest7,
+									                                                                    process_callback,
+									                                                                    this,
+									                                                                    process_internal_object_pool_upload_callback);
```

Comment:
What's the reason behind this change? I tend to find it a bit harder to read personally

Reply:
I think this is an anachronism from early in my development on this branch. I will switch it back to the original way